assets/js/theme/global/quick-search.js

```diff
@@ -40,6 +40,29 @@ export default function () {
             }
 
             $quickSearchResults.html(response);
+            const $quickSearchResultsCurrent = $quickSearchResults.filter(':visible');
+
+            const $noResultsMessage = $quickSearchResultsCurrent.find('.quickSearchMessage');
+            if ($noResultsMessage.length) {
+                $noResultsMessage.attr({
+                    role: 'status',
+                    'aria-live': 'polite',
+                });
+            } else {
+                const $quickSearchAriaMessage = $quickSearchResultsCurrent.closest('nav').find('[data-search-aria-message]');
+                $quickSearchAriaMessage.addClass('u-hidden');
+
+                const predefinedText = $quickSearchAriaMessage.data('search-aria-message-predefined-text');
+                const itemsFoundCount = $quickSearchResultsCurrent.find('.product').length;
+
+                $quickSearchAriaMessage.html('').html(`
```

Comment:
what is the purpose of using html method twice?

Reply:
I need to clear previous search result message before adding the new one

Reply:
Does  previous content won't be [completely replaced](https://api.jquery.com/html/) by the new one from `.html(${itemsFoundCount} ${predefinedText} ${searchQuery})` ?

Reply:
yes it is correct html() deleting previous content by default. removed. thanks!